enews-node/app.js

```diff
@@ -0,0 +1,19 @@
+var NewsFetcher = require('./newsFetcher.js')
+
+var token = process.env.token;
+var newsFetcher = new NewsFetcher(token);
```

Comment:
you might want to do a sanity check that the `process.env.token` value has been set, so that if someone runes this script it's obvious what went wrong. Fail Fast